mods/ra/audio/music.yaml

```diff
@@ -4,6 +4,7 @@ map: Map
 	Hidden: true
 score: Militant Force
 	Hidden: true
+await: Await
```

Comment:
I suggest you move it downward in the `# Aftermath tracks` section so that information isn't lost.